testsuite/ui/multinode/add_node_test.go

```diff
@@ -0,0 +1,99 @@
+// Copyright (C) 2021 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package multinode
+
+import (
+	"testing"
+
+	"github.com/go-rod/rod"
+	"github.com/stretchr/testify/require"
+
+	"storj.io/common/testcontext"
+	"storj.io/storj/private/testplanet"
+	"storj.io/storj/testsuite/ui/uitest"
+)
+
+func TestAddNewNodeButton(t *testing.T) {
+	uitest.Multinode(t, 1, func(t *testing.T, ctx *testcontext.Context, planet *testplanet.Planet, browser *rod.Browser) {
+		startPage := planet.Multinodes[0].ConsoleURL() + "/add-first-node"
+		page := openPage(browser, startPage)
+
+		node := planet.StorageNodes[0]
+		node2 := planet.StorageNodes[1]
+
+		page.MustElement("input#Node\\ ID.headered-input").MustInput(node.ID().String())
+		page.MustElement("input#Public\\ IP\\ Address.headered-input").MustInput(node.Addr())
+		page.MustElement("input#API\\ Key.headered-input").MustInput(node.APIKey())
+
+		addNodeButton := page.MustElement(".add-first-node__left-area__button")
```

Comment:
You can use here `MustElementR(".add-first-node__left-area__button", "Add Node").MustClick()`. and avoid the text check and separate var.

Reply:
Fixed